dev-ruby/net-ssh/net-ssh-6.2.0-r1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_DOCDIR="doc"
+RUBY_FAKEGEM_EXTRADOC="CHANGES.txt README.md THANKS.txt"
+RUBY_FAKEGEM_EXTRAINSTALL="support"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Non-interactive SSH processing in pure Ruby"
+HOMEPAGE="https://github.com/net-ssh/net-ssh"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.rc1.tar.gz -> net-ssh-git-${PV}.tgz"
+
+LICENSE="GPL-2"
+SLOT="$(ver_cut 1)"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
+IUSE="ed25519 test"
+
+MY_P="net-ssh-6.2.0.rc1"
```

Comment:
I have concerns about adding a release candidate for a security-sensitive package. I'd rather wait until there is a released version.

Ah, I see that hashicorp also came to that conclusion: https://github.com/hashicorp/vagrant/commit/034cb8c59ec8a6a46a261af63f1d9925b18c3949

Reply:
Yes so should I backport the patch on the on the 6.1.0 ? @graaff